src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -55,12 +55,14 @@ public void createTable(final String tableName, final boolean overwrite)
     if (overwrite)
     {
       query = "create or replace table identifier(?) (record_metadata " +
-        "variant, record_key variant, record_content variant)";
+        "variant, record_key variant, record_content variant, " +
+        "insert_time timestamp default current_timestamp)";
```

Comment:
Discussion: If we can't set default value in alter table should we also not provide default value on table creation (for consistency)?

Reply:
good point, I think it makes sense to remove the default here for consistency with `alterTable` until snowflake is able to create a default column value on an existing table